### DISCUSSION TITLE
Error: Invalid header: Does not start with Cr24

### DISCUSSION THREAD

**Discussion by 332lfgb6** [quasarframework/quasar#9286] [CLI - Electron mode] [answered]
> Here's my problem

![image](https://user-images.githubusercontent.com/51245335/118363823-c3eb5a80-b5c8-11eb-84a5-9a84f62a572c.png)
> I found an old issule about my problem

https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/2577
![image](https://user-images.githubusercontent.com/51245335/118363889-feed8e00-b5c8-11eb-945a-7b836e04957e.png)
> But I found that my version had surpassed it

![image](https://user-images.githubusercontent.com/51245335/118363956-352b0d80-b5c9-11eb-8d1b-ba1e7492308a.png)
> What should I do next?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@332lfgb6 

Did you try with version "3.1.1" explicitly ( without the ^):
` electron-devtools-installer:"3.1.1"`

**Reply by 332lfgb6** [reply]
I solved this problem by using yarn instead of NPM, and the version that yarn gives me to automatically install is 3.0.0
